Doc/library/sqlite3.rst

```diff
@@ -302,10 +302,10 @@ Module functions
    :type isolation_level: str | None
 
    :param bool check_same_thread:
-       If ``True`` (default), only the creating thread may use the connection.
-       If ``False``, the connection may be shared across multiple threads;
-       if so, write operations should be serialized by the user to avoid data
-       corruption.
+       If ``True`` (default), an exception will be raised if the database connection
```

Comment:
```suggestion
       If ``True`` (default), :exc:`ProgrammingError` will be raised if the database connection
```

Reply:
(I'll reflow before merging, FYI)

Reply:
accepting as is, since it is already better than current text